.pipelines/e2e.yaml

```diff
@@ -3,77 +3,19 @@ trigger: none
 pr:
   branches:
     include:
-    - master
-    - official/*
+      - master
+      - official/*
   paths:
     include:
-    - .pipelines/e2e.yaml
-    - e2e
-    - parts/linux
-    - pkg/agent
+      - .pipelines/e2e.yaml
+      - .pipelines/templates/e2e-template.yaml
+      - e2e
+      - parts/linux
+      - pkg/agent
     exclude:
-    - e2e/windows
-    - pkg/agent/datamodel/sig_config*.go # SIG config changes
-    - pkg/agent/datamodel/*.json # SIG version changes
-    - pkg/agent/testdata/AKSWindows* # Windows test data
-    
-variables:
```

Comment:
not needed anymore ?

Reply:
it's been moved to e2e-template.yaml, so it can be shared between 2 different pipelines